src/Sylius/Behat/Page/Admin/Product/ShowPage.php

```diff
@@ -14,11 +14,32 @@
 namespace Sylius\Behat\Page\Admin\Product;
 
 use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Session;
 use FriendsOfBehat\PageObjectExtension\Page\SymfonyPage;
+use Sylius\Behat\Context\Ui\Admin\Helper\NavigationTrait;
 use Sylius\Component\Core\Model\ProductVariantInterface;
+use Symfony\Component\Routing\RouterInterface;
 
 class ShowPage extends SymfonyPage implements ShowPageInterface
 {
+    use NavigationTrait;
+
+    /**
+     * @template TKey of array-key
+     * @template TValue
+     *
+     * @param array<TKey, TValue>|\ArrayAccess<TKey, TValue> $minkParameters
+     */
```

Comment:
Is this useful at all?

Reply:
I'm not sure 🤷 Static analysis is happy 💃